FirebaseInstallations.podspec

```diff
@@ -49,6 +49,10 @@ Pod::Spec.new do |s|
     s.test_spec 'integration' do |int_tests|
       int_tests.source_files = base_dir + 'Tests/Integration/**/*.[mh]'
       int_tests.resources = base_dir + 'Tests/Resources/**/*'
+      s.pod_target_xcconfig = {
+      'GCC_PREPROCESSOR_DEFINITIONS' =>
+        'FIR_INSTALLATIONS_INTEGRATION_TESTS_REQUIRED=1'
```

Comment:
I'm not following what this does. `pod lib lint` will still try to run all the tests and require a plist to be installed.

Reply:
The flag `FIR_INSTALLATIONS_INTEGRATION_TESTS_REQUIRED` is not set when you do `pod gen ...`, so the tests from this project can be run with no .plist. But yeah, I think, I missed the case of running `pod lib lint` locally without the .plist. Let me think what we can do here...

Reply:
I see. Yep, it's nice to have Command-u work from `pod gen` without configuration. It's probably ok to not worry about the local `pod lib lint` for now.